nipoppy/cli/parser.py

```diff
@@ -148,6 +148,17 @@ def add_arg_verbosity(parser: _ActionsContainer) -> _ActionsContainer:
     return parser
 
 
+def add_arg_keepworkdir(parser: _ActionsContainer) -> _ActionsContainer:
+    """Add a --keep_workdir argument to the parser."""
+    parser.add_argument(
+        "--keep_workdir",
+        type=str,
+        required=False,
+        help="Keep pipeline working directory upon success (default: False).",
```

Comment:
Instead of `(default: False)`, I would put something like `(default: working directory deleted unless a run failed)".

Also ideally mention that this applies to the pyBIDS database too (if it was created)